{"name": "Тест: ввод почты и клик по кнопке «Восстановить»", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF60E81FB05+28789]\n\t(No symbol) [0x00007FF60E7886E0]\n\t(No symbol) [0x00007FF60E62592A]\n\t(No symbol) [0x00007FF60E67930E]\n\t(No symbol) [0x00007FF60E6795FC]\n\t(No symbol) [0x00007FF60E6C28A7]\n\t(No symbol) [0x00007FF60E69F47F]\n\t(No symbol) [0x00007FF60E6BF654]\n\t(No symbol) [0x00007FF60E69F1E3]\n\t(No symbol) [0x00007FF60E66A938]\n\t(No symbol) [0x00007FF60E66BAA1]\n\tGetHandleVerifier [0x00007FF60EB5933D+3410093]\n\tGetHandleVerifier [0x00007FF60EB6E7DD+3497293]\n\tGetHandleVerifier [0x00007FF60EB62A73+3448803]\n\tGetHandleVerifier [0x00007FF60E8E7BBB+848171]\n\t(No symbol) [0x00007FF60E793C3F]\n\t(No symbol) [0x00007FF60E78F6E4]\n\t(No symbol) [0x00007FF60E78F87D]\n\t(No symbol) [0x00007FF60E77ED49]\n\tBaseThreadInitThunk [0x00007FFD25D07374+20]\n\tRtlUserThreadStart [0x00007FFD25F5CC91+33]", "trace": "self = <restore_password_test.TestRestorePassword object at 0x000001BED0C35D10>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"279bc77e88447eaee1e75b5e3800f802\")>\nrecover_password = <pages.login_page.ResetPassword object at 0x000001BED0C74C20>\ntest_user_create = {'email': 'r.shchetnikov:67754004@yandex.ru', 'name': 'Username:67754004', 'password': 'password:67754004'}\n\n    @allure.title(\"Тест: ввод почты и клик по кнопке «Восстановить»\")\n    def test_open_reset_pass_page(self, driver, recover_password, test_user_create):\n        recover_password.click_on_login_to_account()\n        WebDriverWait(driver, 20).until(expected_conditions.element_to_be_clickable(login_page_login_button))\n        recover_password.click_on_recover_password_button()\n        recover_password.input_email_recover(test_user_create[\"email\"])\n        recover_password.click_recover_password_reset_button()\n    \n>       WebDriverWait(driver, 20).until(expected_conditions.visibility_of_element_located(wait_order_number_modal))\n\ntests\\restore_password_test.py:36: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"279bc77e88447eaee1e75b5e3800f802\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x000001BED0877920>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       \tGetHandleVerifier [0x00007FF60E81FB05+28789]\nE       \t(No symbol) [0x00007FF60E7886E0]\nE       \t(No symbol) [0x00007FF60E62592A]\nE       \t(No symbol) [0x00007FF60E67930E]\nE       \t(No symbol) [0x00007FF60E6795FC]\nE       \t(No symbol) [0x00007FF60E6C28A7]\nE       \t(No symbol) [0x00007FF60E69F47F]\nE       \t(No symbol) [0x00007FF60E6BF654]\nE       \t(No symbol) [0x00007FF60E69F1E3]\nE       \t(No symbol) [0x00007FF60E66A938]\nE       \t(No symbol) [0x00007FF60E66BAA1]\nE       \tGetHandleVerifier [0x00007FF60EB5933D+3410093]\nE       \tGetHandleVerifier [0x00007FF60EB6E7DD+3497293]\nE       \tGetHandleVerifier [0x00007FF60EB62A73+3448803]\nE       \tGetHandleVerifier [0x00007FF60E8E7BBB+848171]\nE       \t(No symbol) [0x00007FF60E793C3F]\nE       \t(No symbol) [0x00007FF60E78F6E4]\nE       \t(No symbol) [0x00007FF60E78F87D]\nE       \t(No symbol) [0x00007FF60E77ED49]\nE       \tBaseThreadInitThunk [0x00007FFD25D07374+20]\nE       \tRtlUserThreadStart [0x00007FFD25F5CC91+33]\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:105: TimeoutException"}, "steps": [{"name": "Клик по кнопке Войти в аккаунт", "status": "passed", "start": 1734731298042, "stop": 1734731298088}, {"name": "Клик по кнопке Восстановить пароль", "status": "passed", "start": 1734731298104, "stop": 1734731298139}, {"name": "Ввести почту в поле email", "status": "passed", "parameters": [{"name": "email", "value": "'r.shchetnikov:67754004@yandex.ru'"}], "start": 1734731298139, "stop": 1734731298186}, {"name": "Клик по кнопке Восстановить", "status": "passed", "start": 1734731298186, "stop": 1734731298219}], "start": 1734731298042, "stop": 1734731318531, "uuid": "40d7d36a-3c10-49a8-84d2-529f8c24d5fb", "historyId": "79cfc7485a0fd4718bbc5f98dc26963f", "testCaseId": "79cfc7485a0fd4718bbc5f98dc26963f", "fullName": "tests.restore_password_test.TestRestorePassword#test_open_reset_pass_page", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('recover_password')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "restore_password_test"}, {"name": "subSuite", "value": "TestRestorePassword"}, {"name": "host", "value": "DESKTOP-7LU8FN0"}, {"name": "thread", "value": "21572-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.restore_password_test"}]}